components/QuizCardPage/QuizCard/index.js

```diff
@@ -1,24 +1,18 @@
 import styled from "styled-components";
-import { useState } from "react";
 
-export const questions = [
-  {
-    id: 1,
-    question: "What is the capital of France?",
-    answers: ["Paris", "Berlin", "Madrid", "London"],
-    correctAnswer: "Paris",
-  },
-];
-
-export default function QuizCard() {
-  const [questionsState, setQuestionsState] = useState(questions);
-  return <StyledCard>{questionsState[0].question}</StyledCard>;
+export default function QuizCard({ currentQuestion }) {
+  return (
+    <div>
```

Comment:
Do you really need this `div` here?

If you only display the `question`, I would consider to pass only the `question` as prop and not the entire `currentQuestion` object (because you don't need many parts of this object).

Reply:
No I don't need it, is deleted